microsoft/testsuites/cvm_attestation/cvm_attestation.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import Path
+from typing import Any, Dict
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.testsuite import TestResult
+from lisa.tools import Ls
+from lisa.util import LisaException, SkippedException
+from microsoft.testsuites.cvm_attestation.cvm_attestation_tool import (
+    CVMAttestationTests,
+)
+
+
+@TestSuiteMetadata(
+    area="cvm",
+    category="attestation",
+    description="""
+    This test suite is for generating and verifying CVM attestation report.
+    """,
+)
+class CVMAttestationTestSuite(TestSuite):
+    def before_case(self, log: Logger, **kwargs: Any) -> None:
+        node = kwargs["node"]
+        sev_guest_exists = node.tools[Ls].path_exists(
+            path="/dev/sev-guest",
+            sudo=True,
+        )
+        if not sev_guest_exists:
+            raise LisaException("/dev/sev-guest: Device Not Found")
```

Comment:
this case will always fail in azure platform

Reply:
@LiliDeng 

Correct, This testcase is for nested CVM only. I wanted to fail the testcase if device is not there so we can verify the nested CVM CLH scenarios too.

I have changed exception to SkippedException. Now it wont affect the azure platform.